<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>396</width>
      <height>296</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>351</width>
     <height>276</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QLabel" name="label_sep_addit_numbers">
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>Separate numbers area</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_max_draw">
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>Maximum number of draw</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spinBox_max_addit"/>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_addit_numbers"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_max_addit">
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>Maximum number of additional </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_max_draw"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_addit_numbers">
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>Additional numbers</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_draw_numbers">
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>Draw numbers</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinBox_draw_numbers"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_with_addit">
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>With additional number</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="check_with_addit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_name">
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="text">
       <string>Name of the lotto system</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="check_sep_addit_numbers">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="combo_name">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="edit_name_addition"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_name_addition">
      <property name="text">
       <string>Name of the additional</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
